matrixbot/utils.py

```diff
@@ -108,3 +108,14 @@ def get_command_alias(message, settings):
 def get_aliases(settings):
     res = copy.copy(settings["aliases"])
     return res
+
+
+class MockBot:
```

Comment:
Nice to have a mock server to test the plugin.

Reply:
Actually is very convenient to have these selftest methods that let us test a plugin very quickly without having to deploy it.